core/core.go

```diff
@@ -39,6 +43,37 @@ type Contract struct {
 	ProviderSignature string    `json:"providerSignature"`
 }
 
+// PaymentInfo contains the details describing payment data for a given
+// contract.
+type PaymentInfo struct {
+	// The contract that this payment information is associated with.
+	Contract string `json:"contract"`
+	// Whether or not the renter is currently "paying down" the contract.
+	IsPaying bool `json:"isPaying"`
+	// The date of the last payment.
+	LastPaymentTime time.Time `json:"lastPaymentTime"`
+	// The remaining amount to be paid on the contract.
+	Balance int64 `json:"balance"`
+}
+
+// Transaction describes a transaction involving either a renter or provider.
+type Transaction struct {
```

Comment:
- Why is UserType necessary? 
- What are the units on Amount?
- Typo in "The time the transction occurred."
- Would it be worthwhile to list the sender and recipient in separate fields?

Reply:
- UserType makes things a bit easier to log. It's probably not necessary, though.
- Units on the amount are in tenths of cents
- Thanks :)
- Initially, I would say yes. Now, conceptually, renters transfer funds to contracts, then contracts transfer funds to providers, so it feels a little weird.